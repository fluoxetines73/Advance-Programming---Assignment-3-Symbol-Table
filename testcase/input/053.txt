INSERT x number
ASSIGN x 2b