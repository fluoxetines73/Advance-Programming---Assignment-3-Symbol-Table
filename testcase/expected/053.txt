Invalid: ASSIGN x 2b